From a0b37106329e4da192759982c9068e7ede3356e8 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:00 -0500
Subject: [redhat] kabi: Adding symbol __release_region (kernel/resource.c)

Message-id: <5439061096de3c5cb05feaf20324894b90133547.1418903403.git.poros@redhat.com>
Patchwork-id: 102238
O-Subject: [PATCH RHEL7.1 08/48] kabi: Adding symbol __release_region (kernel/resource.c)
Bugzilla: 1160760
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160760
    https://bugzilla.redhat.com/show_bug.cgi?id=1160760

    Description:
    ============
    These are called by request_mem_region() and
    release_mem_region() macros which are used by
    basically every PCI-driver.
    Used in ioc4_extint.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__release_region
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64le/__release_region
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__release_region
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__release_region

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__release_region b/redhat/kabi/kabi-module/kabi_ppc64/__release_region
new file mode 100644
index 0000000..00b0414
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__release_region
@@ -0,0 +1,2 @@
+#1-
+0x7c61340c __release_region vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/__release_region b/redhat/kabi/kabi-module/kabi_ppc64le/__release_region
new file mode 100644
index 0000000..00b0414
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/__release_region
@@ -0,0 +1,2 @@
+#1-
+0x7c61340c __release_region vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__release_region b/redhat/kabi/kabi-module/kabi_s390x/__release_region
new file mode 100644
index 0000000..00b0414
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__release_region
@@ -0,0 +1,2 @@
+#1-
+0x7c61340c __release_region vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__release_region b/redhat/kabi/kabi-module/kabi_x86_64/__release_region
new file mode 100644
index 0000000..00b0414
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__release_region
@@ -0,0 +1,2 @@
+#1-
+0x7c61340c __release_region vmlinux EXPORT_SYMBOL
-- 
1.7.1